src/java.base/share/classes/jdk/internal/classfile/CodeBuilder.java

```diff
@@ -1192,6 +1234,14 @@ default CodeBuilder lookupswitch(Label defaultTarget, List<SwitchCase> cases) {
         return lookupSwitchInstruction(defaultTarget, cases);
     }
 
+    default CodeBuilder lookupswitch(Consumer<SwitchBuilder> switchHandler) {
+        return block(b -> {
+            var switchBuilder = new SwitchBuilderImpl(b, true);
```

Comment:
The second parameter should’ve probably been `false` here:
```suggestion
            var switchBuilder = new SwitchBuilderImpl(b, /* tableswitch: */ false);
```